Swen/Classes/Swen.swift

```diff
@@ -8,15 +8,11 @@
 
 import Foundation
 
-public protocol Event {
+public protocol EventBase {}
```

Comment:
why did you add EventBase? what is the different of EventBase and Event?
EventBase sounds a little bit odd